Engine/Source/Editor/UnrealEd/Private/CookOnTheFlyServer.cpp

```diff
@@ -5594,7 +5595,16 @@ void UCookOnTheFlyServer::EmptyPackageLists()
 	// We empty these lists because otherwise too many items are cooked
 	PackageTracker->LoadedPackages.Empty();
 	PackageTracker->NewPackages.Empty();
+
+	// We don't to want startup packages
+	CookByTheBookOptions->StartupPackages.Empty();
+
+	//Calling this removes the default startup packages from the internal map in the GRedirectCollector
+	TSet<FName> StartupSoftObjectPackages;
```

Comment:
Why does it say `SoftObject`? Isn't that just a name?

Reply:
I renamed this. It was copied from another part of the engine.